typed_python/types_test.py

```diff
@@ -171,6 +171,18 @@ def pickRandomly(self):
 
 class NativeTypesTests(unittest.TestCase):
 
+    def test_refcount_bug_with_simple_string(self):
+
+        with self.assertRaises(TypeError) as context:
+            _types.refcount(111)
+
+        self.assertTrue(str(context.exception) == "first argument to refcount '111' not a permitted Type", str(context.exception))
```

Comment:
Normally I'd use `self.assertEqual` which produces a nice error message if they are not equal automatically.